ludwig/schema/llms/quantization.py

```diff
@@ -20,6 +21,7 @@ class QuantizationConfig(schema_utils.BaseMarshmallowConfig):
         options=[4, 8],
         default=4,
         description="The quantization level to apply to weights on load.",
+        parameter_metadata=LLM_METADATA["quantization"]["bits"],
```

Comment:
will you add more for the other parameters?

Reply:
The others are low impact, so wanted to leave them as-is for now.

Reply:
ah gotcha